compiler/ilgen/OMRIlGeneratorMethodDetails.hpp

```diff
@@ -38,29 +38,17 @@ namespace OMR { typedef OMR::IlGeneratorMethodDetails IlGeneratorMethodDetailsCo
 class TR_FrontEnd;
 class TR_ResolvedMethod;
 namespace TR { class IlGeneratorMethodDetails; }
+namespace TR { class IlVerifier; }
 
 namespace OMR
 {
 
 /**
- * IlGeneratorMethodDetailsBase defines the IlGeneratorMethodDetails API that common code can count on.  A front
- *   end will typically extend this class (perhaps even with a hierarchy of classes) and then map its own concrete
- *   base class onto OMR::IlGeneratorMethodDetails.  In this way, common code can count on a particular API
- *   existing, but IlGeneratorMethodDetails will be passed around with all front-end specific data and API intact
- *   (so less up/down casting).
+ * Accessing ANY language-pecific API/data in common code is prohibited.
  *
- * Accessing ANY front-end specific API/data in common code is prohibited.
- *
- * IlGeneratorMethodDetailsBase defines the IlGenRequest API that common code can count on, although it's more documentation
- *   then enforcement.  A front end will typically extend this class (perhaps even with a hierarchy of classes)
- *   and then map its own concrete base class onto OMR::IlGeneratorMethodDetailsBase.
- *
- * Accessing ANY front-end specific API/data in common code is prohibited since other front end builds would fail.
- *
- * Note all functions are non-virtual in this class, so declarations should never use the IlGeneratorMethodDetailsBase type
- *   directly: all variable declarations should use the OMR::IlGeneratorMethodDetails type instead.
+ * IlGeneratorMethodDetails defines the IlGenRequest API that common code
+ * can count on.
```

Comment:
I realize you're just truncating the comment text that was already there, but I really don't like the folksy "can count on" text.  I'd say delete "that common code can count on" as it provides no value here.